### DISCUSSION TITLE
How are the materialmkdocs' screenshots made?

### DISCUSSION THREAD

**Discussion by couleurm** [squidfunk/mkdocs-material#6619] [Q&A] [answered]
they look gorgeous with their little drop shadow, how do you make them squid?

![](https://squidfunk.github.io/mkdocs-material/assets/screenshots/navigation-tabs-sticky.png)

**Comment by squidfunk** [comment, marked as answer]
https://chromewebstore.google.com/detail/standardized-screenshot/pabdhaakclnechgfhmnhkcbmjobeoope
